src/embed/connection.rs

```diff
@@ -69,7 +69,13 @@ async fn connection_future<'a, 'b>(
         "beginSync" => execute(ctx, do_begin_sync, req).await,
         "maybeEndSync" => execute(ctx, do_maybe_end_sync, req).await,
         "close" => {
-            req.response.send(Ok("".into())).await;
+            let result = ctx
```

Comment:
I think you may want to swap the order of these two calls. You are closing the idb store *then* stopping the message loop that is using it. It seems like this could result in a few stray calls still getting through and running on the closed store.

Reply:
Which two calls? Are you saying `send` the result before calling `close`?

Reply:
I'm an idiot, nevermind.